india_compliance/install.py

```diff
@@ -10,6 +10,7 @@
 POST_INSTALL_PATCHES = (
     ## ERPNext
     "setup_custom_fields_for_gst",
+    "add_hsn_to_item_variant_copy_fields",
```

Comment:
EPRNext Patches are old patches upto v13 kept for users upgrading from older versions to v14.

Add this patch under India Compliance (as commented)

Reply:
Have changed the ordering